proxy.go

```diff
@@ -158,34 +140,38 @@ func (proxy *proxy) allocateTokens(vm *vm, numIOStreams int) (*api.IOResponse, e
 	}, nil
 }
 
-func (proxy *proxy) claimToken(token Token) (*tokenInfo, error) {
+func (proxy *proxy) claimToken(token Token) (*ioSession, error) {
```

Comment:
Although it might not strictly be part of this PR, should this function check for `token == nilToken` as `releaseToken()` does?